plugins/scaffolder-react/src/next/components/Stepper/Stepper.tsx

```diff
@@ -216,10 +217,14 @@ export const Stepper = (stepperProps: StepperProps) => {
 
   const mergedUiSchema = merge({}, propUiSchema, currentStep?.uiSchema);
 
-  const handleCreate = useCallback(() => {
-    props.onCreate(stepsState);
-    analytics.captureEvent('click', `${createLabel}`);
-  }, [props, stepsState, analytics, createLabel]);
+  const handleCreate = useDebouncedCallback(
+    () => {
+      props.onCreate(stepsState);
+      analytics.captureEvent('click', `${createLabel}`);
+    },
+    [props, stepsState, analytics, createLabel],
+    250,
```

Comment:
Is it possible to disable the button until the new page loads or something? Just checking. Instead of having a fixed time, I mean. Maybe this is good enough.

Reply:
Indeed, I'm then passing `isCreating` from `TemplateWizardPage` then.

Reply:
<img width="443" alt="image" src="https://github.com/user-attachments/assets/f696e125-2f9e-4232-a422-fa349f7e797e" />

<img width="553" alt="image" src="https://github.com/user-attachments/assets/d2c619b2-2115-4de3-9368-fc1b08adfcc3" />

```log
Stepper › state tracking › should render perfectly when using field extensions that may do some strange things

    expect(jest.fn()).toHaveBeenCalledTimes(expected)

    Expected number of calls: 1
    Received number of calls: 3
```

@freben Any advice for testing?